External authorizer user tutorial (#649)
kyma-project/istio: docs/user/tutorials/01-10-external-authorization-provider.md

## Prerequisites

* Kyma installation with the Istio module enabled. If you use a Kyma domain, also the API Gateway module must be enabled.

> * Set up [your custom domain](https://kyma-project.io/#/api-gateway/user/tutorials/01-10-setup-custom-domain-for-workload) or use a Kyma domain instead.

Reviewer: ```suggestion
* Prepare a domain you want to use and export its name as an environment variable:
    ```
    export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
    ```
```

## Steps

> ### Create a workload

Reviewer: ```suggestion
### Create a Workload
```

> 1. Export the namespace where you want to deploy the HTTPBin Service:

Reviewer: ```suggestion
1. Export the name of the namespace in which you want to deploy the HTTPBin Service:
```

    ```
    export NAMESPACE={NAMESPACE_NAME}
    ```

> 2. Create a namespace with istio-injection enabled and deploy the HTTPBin:

Reviewer: ```suggestion
2. Create a namespace with Istio injection enabled and deploy the HTTPBin Service:
```
